core/primitives/src/utils.rs

```diff
@@ -211,6 +211,27 @@ macro_rules! unwrap_or_return {
     };
 }
 
+/// Macro to either return value if the result is Some, or exit function logging error.
+#[macro_export]
+macro_rules! unwrap_obj_or_return {
```

Comment:
how is this different from the previous macro?
at least comment is the same, but you are not logging.

Reply:
One is unwrapping results the other is unwrapping object. I can log in the other.

Reply:
`unwrap_option_or_return` would be less confusing I guess

Reply:
Totally, when coding it, I confused `Option` with `Object`.